src/main/kotlin/com/github/mitchellolsthoorn/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.mitchellolsthoorn.testgenie.services
+
+import com.intellij.ide.highlighter.JavaFileType
+import com.intellij.openapi.project.Project
+import com.intellij.psi.JavaCodeFragmentFactory
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.ui.EditorTextField
+import com.intellij.ui.components.JBScrollPane
+import java.awt.BorderLayout
+import java.awt.Dimension
+import javax.swing.*
+
+class TestCaseDisplayService(private val project: Project) {
+
+    val mainPanel: JPanel = JPanel()
+    private val applyButton: JButton = JButton("Apply")
+    private val allTestCasePanel: JPanel = JPanel()
+    private val scrollPane: JBScrollPane = JBScrollPane(allTestCasePanel)
+
+    init {
+        allTestCasePanel.layout = BoxLayout(allTestCasePanel, BoxLayout.Y_AXIS)
```

Comment:
I noticed that each test that is displayed has a lot of newline entries which results in empty space and a lot to scroll. Is this on purpose? Is it possible to reduce this space?

Reply:
Maybe we should also change the color of the box that was selected. So a selected box becomes blue or something to make it clearer. But I am not sure if that will be a good idea.

Reply:
This has already been addressed in one of the later commits - did you pull?